.gitignore

```diff
@@ -12,3 +12,6 @@
 /shared-*
 /release-*
 /bin
+/nvidia-cdi-hook
```

Comment:
nit: could we replace the `nvidia-*` binaries with a `/nvidia-*` rule?

Reply:
Done

Reply:
We can remove this line as well.

Reply:
removed